### PR TITLE
Redesign/partners

### PR DESCRIPTION
**Pull request by MichaelBooi**
- Added container
- Downloaded images
- Bold title
- Placement images
- Links on images
- Small, medium and large column sizes added
- Added if statement to the js fuction for the courses slide

Had some trouble with this but in the end we got it working however the image sizes are very different compared to each other. Should I edit the image size via photoshop, create a new class per image or do you have the partner pictures stored somewhere? :)
screenshot from 2016-10-27 15-13-52

### CODE REVIEW

File: app/assets/images/partners/rooskookt-logo.svg [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+			c0,12.654,7.278,20.269,15.453,20.269c7.503,0,14.895-6.495,14.895-19.373c0-13.998-7.839-20.493-15.23-20.493
+			C607.142,63.322,599.863,70.377,599.863,82.92z"/>
+		<path d="M659.325,34.318c0.112,0.784-0.336,1.344-0.56,1.68c-0.336,0.448-0.448,0.784-0.448,1.231v43.898l25.645-23.741
+			c2.016,0.448,3.92,0.56,5.936,0.56h2.911l-21.612,20.27l25.196,30.684h-0.784c-5.375,0-9.631,0.336-9.631,0.336l-20.94-26.093
+			l-6.72,6.383V108.9h-8.175V34.318H659.325z"/>
+		<path d="M728.643,42.493c0.336,1.12-0.447,1.792-0.672,3.024c-0.56,2.799-1.231,11.982-1.231,11.982h16.686v6.719h-16.798
+			c-0.672,6.719-1.008,13.326-1.008,20.045c0,7.391,0.336,9.406,0.336,9.854c0.784,6.159,4.815,8.175,8.623,8.175
+			c3.136,0,7.055-1.344,11.423-4.592l2.463,6.384c-4.367,3.136-10.302,5.711-16.35,5.711c-5.263,0-9.295-2.016-11.646-5.487
+			c-2.911-4.255-3.023-9.295-3.023-13.886c0-8.735,0.448-17.47,1.12-26.204h-11.982l0.112-6.607H718.9l0.896-13.662L728.643,42.493z
+			"/>
```
Comment (miriamtocino) [inline app/assets/images/partners/rooskookt-logo.svg]: It looks like there it something weird happening with your tabs in this file. They are HUGE :smile:
Can you check if you can turn them to span only 2 spaces as we spoke before?

File: app/assets/javascripts/frontend/courses.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ function currentSlide(n) {
 }
 
 function showSlides(n) {
+  if ($('.mySlides').length > 0)
```
Comment (miriamtocino) [inline app/assets/javascripts/frontend/courses.js]: How is this related to this pull?

File: config/environments/development.rb.save [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  config.hosts = {
+    en: 'www.developmentbootcamp.com.dev',
+    nl: 'www.developmentbootcamp.nl.dev'
+  }
+
+  config.facebook_id = {
+    en: '1706631842883019',
+    nl: '872347686220061'
+  }
+end
```
Comment (miriamtocino) [inline config/environments/development.rb.save]: You should remove this file from the commit. The extension `.save` doesn't look good to me...

File: app/assets/javascripts/frontend/courses.js (hunk 2 above)
Comment (MichaelBooi) [inline app/assets/javascripts/frontend/courses.js]: Michal asked me to fix this so I did this in the same branch, should I do this in another branch next time?
